Lib/sysconfig.py

```diff
@@ -201,6 +201,10 @@ def joinuser(*args):
         return joinuser("~", "Library", sys._framework,
                         "%d.%d" % sys.version_info[:2])
 
+    # VxWorks has no home directories
+    if sys.platform == "vxworks":
+        return sys.prefix
```

Comment:
Would you mind to move it after "if env_base.." and before "def joinuser..." to clarify that the joinuser() function makes no sense on VxWorks?

Same remark for site.py which has copy of this function.

Reply:
Modified.